src/peergos/server/util/PeergosNetworkUtils.java

```diff
@@ -26,7 +26,7 @@
 public class PeergosNetworkUtils {
 
     public static String generateUsername(Random random) {
-        return "username_" + Math.abs(random.nextInt() % 1_000_000);
+        return "username_" + Math.abs(random.nextInt() % 1_000_000_000);
```

Comment:
Why this number?

Reply:
Because it's bigger, round, and less than max int.